chart/pod-reaper/templates/rbac.yaml

```diff
@@ -16,6 +16,9 @@ rules:
 - apiGroups: [""]
   resources: ["pods"]
   verbs: ["list", "delete"]
+- apiGroups: [""]
+  resources: ["pods/eviction"]
+  verbs: ["post"]
```

Comment:
I tried to get this working locally and it didn't work. After some troubleshooting I switched this over to "create" instead of "post". I should make a note to lookup why k8s didn't use http-verbs here :)

Reply:
Fixed and pushed.  Thanks for testing this, I don't have much experience with RBAC